wrangler.toml.example

```diff
@@ -1,11 +1,10 @@
 name = "cf-metrics"
```

Comment:
I propose finding a more descriptive identifier for this worker. I find "cf-metrics" too vague/generic.

ATM, this worker is observing access tokens and reporting the usage data. In the future, we will also want to allow access to authenticated clients only. Also, this worker is specific to spark-stats service.

Here are some alternatives I find more fitting:

- spark-stats-auth
- spark-stats-interceptor
- spark-stats-handler

@juliangruber do you have any thoughts on this?

Reply:
I also find `cf-metrics` too generic. Once we have found a name, let's update the repository name to match.

> In the future, we will also want to allow access to authenticated clients only

I'm not sure if that isn't better handled in `spark-stats`, so I'd be cautious to count this as a requirement. To me, its sole purpose is to report api token usage to InfluxDB. What about

- `spark-stats-request-metrics`
- `spark-stats-user-metrics`

Reply:
@juliangruber `spark-stats-request-metrics` sound good to me. Miro do you agree?

Reply:
ping @bajtos

Reply:
> > In the future, we will also want to allow access to authenticated clients only
> 
> I'm not sure if that isn't better handled in `spark-stats`, so I'd be cautious to count this as a requirement. To me, its sole purpose is to report api token usage to InfluxDB. What about

In my understanding, we must do authentication at the edge (in a CF worker) because we are heavily caching the responses, and therefore, most requests never reach spark-stats.

Having written that, I am fine with leaving authentication out of the scope of the current discussion.

The name `spark-stats-request-metrics` sounds good to me.